src/tv2-common/content/server.ts

```diff
@@ -149,24 +149,30 @@ function GetServerTimeline(
 		}),
 
 		...(adLibPix
-			? config.stickyLayers.map<TSR.TimelineObjSisyfosChannel & TimelineBlueprintExt>(layer => {
-					return literal<TSR.TimelineObjSisyfosChannel & TimelineBlueprintExt>({
+			? [
+					literal<TSR.TimelineObjSisyfosChannels & TimelineBlueprintExt>({
 						id: '',
 						enable: {
-							while: serverEnableClass
+							start: 1
 						},
 						priority: 1,
-						layer,
+						layer: sourceLayers.Sisyfos.SisyfosPersistedLevels,
 						content: {
 							deviceType: TSR.DeviceType.SISYFOS,
-							type: TSR.TimelineContentTypeSisyfos.CHANNEL,
-							isPgm: 0
+							type: TSR.TimelineContentTypeSisyfos.CHANNELS,
+							overridePriority: 1,
+							channels: config.stickyLayers.map<TSR.TimelineObjSisyfosChannels['content']['channels'][0]>(layer => {
```

Comment:
Just a FYI: StickyLayers has been removed on develop in the refactor for "acceptPersist" and "wantsToPersist" for Audio levels